themes/StarterTheme/templates/catalog/_partials/active_filters.tpl

```diff
@@ -0,0 +1,10 @@
+{if $activeFilters|count}
+  <section class="active_filters">
+  <h1 class="h3">{l s='Active Filters'}</h1>
```

Comment:
Hi @inetbiz,

Would you mind making a PR with your suggestions, please ?

Regards